exporter/opentelemetry-exporter-prometheus/src/opentelemetry/exporter/prometheus/__init__.py

```diff
@@ -157,7 +158,10 @@ def _translate_to_prometheus(
         label_keys = []
         for key, value in metric.attributes.items():
             label_keys.append(self._sanitize(key))
-            label_values.append(str(value))
+            if isinstance(value, (list, tuple)):
```

Comment:
I'd recommend checking if it's a Sequence, but you have to make sure it's not a str first which implements Sequence: https://github.com/open-telemetry/opentelemetry-python/blob/2098c5644588cd4e766eb8e31a1ca26c4e00ea46/opentelemetry-api/src/opentelemetry/util/types.py#L18-L27

Reply:
Haha, yes, I did use `Sequence` first and exactly that happened to me. Will refactor :+1:

Reply:
Fixed